src/test/java/uk/gov/hmcts/reform/em/stitching/service/impl/DocumentConversionServiceImplTest.java

```diff
@@ -39,18 +41,18 @@ public void setup() {
         );
     }
 
-    @Test(expected = IOException.class)
-    public void noHandler() throws IOException {
+    @Test
+    void noHandler() {
```

Comment:
Could we make the test name pertinent to what this will test? I.e. throwing a IOException.